Tests/test_configuration_checker.py

```diff
@@ -3,7 +3,7 @@
 from kalliope.core.ConfigurationManager.ConfigurationChecker import ConfigurationChecker, NoSynapeName, \
                                                 NoSynapeNeurons, NoSynapeSignals, NoValidSignal, MultipleSameSynapseName
 from kalliope.core.Models import Synapse
-from kalliope.core.Utils.Utils import ModuleNotFoundError
+from kalliope.core.Utils.Utils import KalliopeModuleNotFoundError
```

Comment:
Why did you rename this Error ? I do not understand why adding `Kalliope` is relevant here ?

Reply:
This name is also used in a lot of library and was a little bit confusing